changeFeedSamples/trackingChangesToBlobs/src/main/java/ChangeFeedTimer.java

```diff
@@ -0,0 +1,171 @@
+import com.azure.storage.blob.BlobClient;
+import com.azure.storage.blob.BlobContainerClient;
+import com.azure.storage.blob.BlobServiceClient;
+import com.azure.storage.blob.BlobServiceClientBuilder;
+import com.azure.storage.blob.changefeed.BlobChangefeedClient;
+import com.azure.storage.blob.changefeed.BlobChangefeedClientBuilder;
+import com.azure.storage.blob.changefeed.BlobChangefeedPagedIterable;
+import com.azure.storage.blob.changefeed.BlobChangefeedPagedResponse;
+import com.azure.storage.blob.changefeed.models.BlobChangefeedEvent;
+import com.azure.storage.blob.specialized.BlockBlobClient;
+import com.azure.storage.common.StorageSharedKeyCredential;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Properties;
+import java.util.Timer;
+import java.util.TimerTask;
+import java.util.function.Predicate;
+import java.util.stream.Stream;
+
+public class ChangeFeedTimer {
+    /**
+     * Sets up a timer that checks for new events that are filtered based on preferences in TimerHelper. Saves cursor to
+     * blob in given storage account for future use
+     */
+    public static void main(String[] args) throws IOException {
+        String storingCursorContainer = "storingcursorcontainer";
+        String blobCursorName = "storingcursorblob";
```

Comment:
Similar feedback as on .NET PR. This could go to config/constants. Other thing is that can we use nouns in the names like "cursorStorage" or so ?